clr.nativebuild.props

```diff
@@ -0,0 +1,7 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CL_MPCount>$(NumberOfEnabledCore)</CL_MPCount>
+    <ProcessorNumber>$(CL_MPCount)</ProcessorNumber>
+    <MultiProcessorCompilation>true</MultiProcessorCompilation>
```

Comment:
I'm not sure that $(MultiProcessorCompilation) is a property? In the C++ build I see this going to CL
`MultiProcessorCompilation="%(ClCompile.MultiProcessorCompilation)"`
that means that it is item metadata. It has to either be set on ClCompile items individually, or on all of them using ItemDefinitionGroup.

$(CL_MPCount) is what does the trick I think -- it is converted to metadata
```
<ProcessorNumber Condition="'$(CL_MPCount)'                              != ''">$(CL_MPCount)</ProcessorNumber>
...
<CL
ProcessorNumber="%(ClCompile.ProcessorNumber)"
```

and I guess that setting ProcessorNumber (/MP:n) implies MultiprocessorCompilation (/MP).

Sicne CL_MPCount sets ProcessorNumber, I think only CL_MPCount is required, of these three.

Reply:
I got all this info by doing "msbuild /pp" on a vcxproj.

Reply:
Yeah, confirm CL_MPCount is required and sufficient to control number of CL instances. Setting MultiProcessorCompilation to false can be overridden by CL_MPCount > 1.

Reply:
You could even pass it with /p globally instead of using this file. But using a file like this means that individual projects can modify the value themselves if they want.